manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -124,9 +124,12 @@ const toPrivateOnlySignTest = async () => {
     const initalPrivateBalance = await privateWallet.getPrivateBalance(assetId);
     console.log("The inital private balance is: ", initalPrivateBalance.toString());
 
-    const signResult = await privateWallet.toPrivateSend(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
+    const signResult = await privateWallet.toPrivateBuild(assetId, amount, polkadotConfig.polkadotSigner, polkadotConfig.polkadotAddress);
 
-    console.log("The result of the signing: ", signResult);
+    console.log("The result of the signing: \"0x", JSON.stringify(signResult));
+
+    // remove first 3 bytes of the signResult
+    console.log("For xcm remote transact payload, please use: \"0x" + JSON.stringify(signResult.txs).slice(10));
```

Comment:
why slice 10 if you're removing 3

Reply:
this string is like this "0xaabbcc...." including the ". So that's 9 characters and an escape character.

Reply:
if you remove the first 3 the string will be `aabbcc...."` including the trailing `"`. Is that fine?
Or does your slice also remove the trailing `"`

Reply:
If you remove from front and back, better use https://stackoverflow.com/a/952945
more robust if this string ever changes in length, however unlikely

Reply:
ok, seems i misunderstood, the `signResult` doesn't actually need trimming from the front, you just take the first 10 chars and put them in the log prepended with `0x`

But then the comment in 131 is wrong, no?

Reply:
It does need to be trimmed from the front, without the 0x. In the actual payload 2 characters are a byte.
Since i call stringify i also need to remove the quotation mark and re-add it.

Reply:
still dont get it then.
`JSON.stringify(signResult.txs).slice(10));` simply returns the first 10 chars = 5 stringified bytes

Reply:
JSON.stringify(signResult.txs):
```
["0xb908042800042f0000010100000000000000000000000000000000000000000000000000000000000000040000e8890423c78a000000000000000000040000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000057ab1057c99f9c06746da802cff6a6f7e91546ea4f84131c8fab82bacb8f501c08623bf38da16253a7cbcf6c69f33f1134c55aea4f25526b4e15291f414672af89259940ff384cafaa0e62892b9bec33120a37992631da2ccfa93d2e9d91af17237c8b2ad9418f64914178219610d7b306ecab4c303f0db5ffc4240ad044b4cc12d6ff850d296a4caf658f54eeb32d7401295a845b73094230b2a1b3df8ca81619c84ff496459892cacfbea92a11febb3ce82bff3b479c1803ae55a9acc8afae17623bf38da16253a7cbcf6c69f33f1134c55aea4f25526b4e15291f414672af8958497ff0c7274eab384c33cbd646332d51b95a3a6d9e24a3dbc7d3b4895ce1fdfd5216072bde990a3413d0b0db11cab8dd0e4aaa73398e7d0d35a237a1228da3c884eb32ad9755fe92fda1945e280aa18f79a889dd89a39ae8c0f86d70f54a1c000bbd3dacf095ba4a2c46590d891d05bd14a8000a688e0d1dc866750ed43a8896efd13eadbb2bd65342020659c7a6130ae315ed7d3b74a656d7136087716b68155b240e4804651dfd300f682932e5dda5a4d2bc414687de1a925ac49452ef122f709473ba7df9f604fde9b205a93d991a83ac9fe26000988e56d8932e40bb9f04"]
```

JSON.stringify(signResult.txs).slice(10)):
For xcm remote transact payload, please use: "0x2800042f0000010100000000000000000000000000000000000000000000000000000000000000040000e8890423c78a000000000000000000040000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000057ab1057c99f9c06746da802cff6a6f7e91546ea4f84131c8fab82bacb8f501c08623bf38da16253a7cbcf6c69f33f1134c55aea4f25526b4e15291f414672af89259940ff384cafaa0e62892b9bec33120a37992631da2ccfa93d2e9d91af17237c8b2ad9418f64914178219610d7b306ecab4c303f0db5ffc4240ad044b4cc12d6ff850d296a4caf658f54eeb32d7401295a845b73094230b2a1b3df8ca81619c84ff496459892cacfbea92a11febb3ce82bff3b479c1803ae55a9acc8afae17623bf38da16253a7cbcf6c69f33f1134c55aea4f25526b4e15291f414672af8958497ff0c7274eab384c33cbd646332d51b95a3a6d9e24a3dbc7d3b4895ce1fdfd5216072bde990a3413d0b0db11cab8dd0e4aaa73398e7d0d35a237a1228da3c884eb32ad9755fe92fda1945e280aa18f79a889dd89a39ae8c0f86d70f54a1c000bbd3dacf095ba4a2c46590d891d05bd14a8000a688e0d1dc866750ed43a8896efd13eadbb2bd65342020659c7a6130ae315ed7d3b74a656d7136087716b68155b240e4804651dfd300f682932e5dda5a4d2bc414687de1a925ac49452ef122f709473ba7df9f604fde9b205a93d991a83ac9fe26000988e56d8932e40bb9f04"]

Reply:
I guess there's a [ missing at the start though. Actually it would be best to return these payloads without any brackets, quotations or json-ification stuff.

Reply:
As discussed i'll do this clean up in follow up https://github.com/Manta-Network/sdk/issues/70